RELEASE_NOTES.md

```diff
@@ -1,7 +1,12 @@
 # Release Notes
 
-## stable-1.15-1.5.0, stable-1.16-1.5.0
+## stable-1.15-1.6.0, stable-1.16-1.6.0
+
+* \[dex-k8s-authenticator\]: Now supports a kubectl credentials plugin for automatically managing identity tokens. Instructions for downloading the plugin and configuring kubectl can be found at `https://\u003ccluster-ip\u003e/token/plugin`. ([#212](https://github.com/mesosphere/kubernetes-base-addons/pull/212), [@jr0d](https://github.com/jr0d))
```

Comment:
accidental encoding in the link?` \u003e`

Reply:
```suggestion
* \[dex-k8s-authenticator\]: Now supports a kubectl credentials plugin for automatically managing identity tokens. Instructions for downloading the plugin and configuring kubectl can be found at `https://<cluster-ip>/token/plugin`. ([#212](https://github.com/mesosphere/kubernetes-base-addons/pull/212), [@jr0d](https://github.com/jr0d))
```